### PR TITLE
PR: Add '+' button to open new notebooks.

### LINKED ISSUE #11: Add a "+" button to the tab bar to let people know how to open new notebooks
This button is similar to the one found in Firefox, e.g.

seleccion_001

### PR DESCRIPTION
**Pull request by dalthviz**
Fixes #11

### CODE REVIEW

**Comment by ccordoba12** [comment]
This is so nice! Thanks a lot for it.

Is the notebook working well for you on Windows?

File: spyder_notebook/notebookplugin.py [hunk 1]
```diff
@@ -57,7 +57,13 @@ def __init__(self, parent):
         self.initialize_plugin()
 
         layout = QVBoxLayout()
-        self.tabwidget = Tabs(self, self.menu_actions)
+        new_notebook_btn = create_toolbutton(self,
```
Comment (ccordoba12) [inline spyder_notebook/notebookplugin.py]: Please add a blank line before this one.

File: spyder_notebook/notebookplugin.py [hunk 2]
```diff
@@ -57,7 +57,13 @@ def __init__(self, parent):
...
 
         layout = QVBoxLayout()
-        self.tabwidget = Tabs(self, self.menu_actions)
+        new_notebook_btn = create_toolbutton(self,
+                                             icon=ima.icon('project_expanded'),
+                                             tip=_('Open a new notebook'),
+                                             triggered=self.create_new_client)
+        corner_widgets = {Qt.TopRightCorner: [new_notebook_btn]}
+        self.tabwidget = Tabs(self, actions=self.menu_actions,
+                              corner_widgets=corner_widgets)
```
Comment (ccordoba12) [inline spyder_notebook/notebookplugin.py]: Also add a blank line after this one.

**Comment by dalthviz** [comment]
@ccordoba12 about how the notebook is working:

1. Sometimes Spyder fails to initialice the notebook server so the plugin stays in the load page (`Connecting to kernel`), so you have to open a new notebook to make it work (I think that maybe this is a problem of synchronization where the notebook server isn't running when the first notebook is trying to run in Spyder). 

2. Also if you close Spyder and the `cmd` where the server is running (the one open by Spyder for the first time), you will have to manually start the notebook server with `jupyter notebook` command because Spyder doesn't start it again when you reopen Spyder. 

3. Also, another problem is the update of the font. When I was trying to change the language a traceback appear:

```
Traceback (most recent call last):
  File "C:\Users\Daniel\Anaconda3\lib\site-packages\spyder\plugins\configdialog.py", line 218, in button_clicked
    configpage.apply_changes()
  File "C:\Users\Daniel\Anaconda3\lib\site-packages\spyder\plugins\configdialog.py", line 97, in apply_changes
    self.save_to_conf()
  File "C:\Users\Daniel\Anaconda3\lib\site-packages\spyder\plugins\configdialog.py", line 410, in save_to_conf
    self.set_font(font, option)
  File "C:\Users\Daniel\Anaconda3\lib\site-packages\spyder\plugins\configdialog.py", line 985, in set_font
    plugin.update_font()
  File "C:\Users\Daniel\.spyder-py3\plugins\spyder_notebook\notebookplugin.py", line 92, in update_font
    client.set_font(font)
AttributeError: 'NotebookClient' object has no attribute 'set_font'
```

4. Another traceback appear also when trying to undock the plugin in Spyder 4.0.0.dev0:

```
Traceback (most recent call last):
  File "C:\Users\Daniel\.spyder-py3\plugins\spyder_notebook\notebookplugin.py", line 131, in refresh_plugin
    self.update_plugin_title.emit()
AttributeError: 'function' object has no attribute 'emit'
Using existing server at C:\Users\Daniel
Traceback (most recent call last):
  File "C:\Users\Daniel\Desktop\Spyder\spyder\spyder\api\plugins.py", line 141, in visibility_changed
    self.refresh_plugin()   # To give focus to the plugin's widget
  File "C:\Users\Daniel\.spyder-py3\plugins\spyder_notebook\notebookplugin.py", line 131, in refresh_plugin
    self.update_plugin_title.emit()
AttributeError: 'function' object has no attribute 'emit'
Traceback (most recent call last):
  File "C:\Users\Daniel\Desktop\Spyder\spyder\spyder\api\plugins.py", line 141, in visibility_changed
    self.refresh_plugin()   # To give focus to the plugin's widget
  File "C:\Users\Daniel\.spyder-py3\plugins\spyder_notebook\notebookplugin.py", line 131, in refresh_plugin
    self.update_plugin_title.emit()
AttributeError: 'function' object has no attribute 'emit'
```

5. Also a message appears in the notebook pointing an error, however maybe this is normal since the notebook is at first temporal:

error_notebook

**Comment by ccordoba12** [comment]
About your comments:

1. Please play with the `range` value below this line:

    https://github.com/spyder-ide/spyder-notebook/blob/master/spyder_notebook/utils/nbopen.py#L57

    and submit a PR when you find an appropriate value.

2. We'll fix this in the future, i.e. we'll hide the server cmd window by default. For now, I think it's worth to have it around to check for errors.

3. Fixed in PR #17.

4. Fixed in master.

5. I have no idea about that error. We'll have to check it when we meet in person.

**Comment by ccordoba12** [comment]
@dalthviz, please open an issue about your point 5. so we don't forget to fix it.

**Pull request merged**